### PR TITLE
Add version option for e2e-bootstrap

### PR DESCRIPTION
**Pull request by lizhuqi**
**What this PR does / why we need it**:
Add version option to e2e bootstrap. This adds the flexibility to install different release of the driver for e2e test.  

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,6 +18,7 @@ REGISTRY ?= andyzhangx
 REGISTRY_NAME = $(shell echo $(REGISTRY) | sed "s/.azurecr.io//g")
 IMAGE_NAME ?= smb-csi
 IMAGE_VERSION ?= v1.0.0
+VERSION ?= $(IMAGE_VERSION)
```
Comment (andyzhangx) [inline Makefile:21]: by default, it should be `latest`

**Review by andyzhangx — COMMENTED** [review]
/ok-to-test

File: Makefile (hunk 1 above)
Comment (lizhuqi) [reply Makefile]: Thanks! Updated to latest.

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Pull request merged**